packages/design-system/src/components/TextField/useLabelMask.test.tsx

```diff
@@ -10,6 +10,7 @@ import useLabelMask, {
 } from './useLabelMask';
 import { render, screen } from '@testing-library/react';
 import { act, renderHook } from '@testing-library/react-hooks';
+// import { act, renderHook } from '@testing-library/preact-hooks';
```

Comment:
i'm not sure how to dynamically switch between imports. 

preact-hooks are needed to test when in preact mode and react-hooks needed for react mode.